drivers/pinctrl/pfc_rcar.c

```diff
@@ -189,6 +246,10 @@ int pinctrl_configure_pin(const pinctrl_soc_pin_t *pin)
 		return -EINVAL;
 	}
 
+	if (pin->voltage != PIN_VOLTAGE_NONE) {
+		pfc_rcar_set_voltage(pin->pin, pin->voltage);
```

Comment:
for some of platforms we don't have implementation of `pfc_rcar_get_io_voltage_regs` and it can be one of the reason why it has to return error code, but here we haven't to do anything with the error status, but maybe somebody will need it in the future, I can change ret type from `int` to `void`

Reply:
done